users/validation.py

```diff
@@ -0,0 +1,13 @@
+import re
+
+from django.core.exceptions import ValidationError
+
+def email_check(value):
+    if re.match('^([a-z0-9_\.-]+)@([\da-z\.-]+)\.([a-z\.]{2,6})$', value) is None:
```

Comment:
true or false를 판단할 수 있는 경우에는 굳이 `is None`과 같이 비교할 필요가 없습니다. 아래에 작성하신 `password_check`와 같이 작성하시면 되겠습니다!
```suggestion
    if not re.match('^([a-z0-9_\.-]+)@([\da-z\.-]+)\.([a-z\.]{2,6})$', value):
```